src/public/modules/forms/admin/controllers/activate-form-modal.client.controller.js

```diff
@@ -119,24 +118,6 @@ function ActivateFormController(
       // CorpPass doesn't return any error page even with the wrong e-service id
```

Comment:
if the endpoint is not called for CP forms, why is there a switch case for CP forms in server controller handleValidateFormEsrvcId function?

Reply:
the switch case isn't called from the frontend but for the old backend endpoint, it accepts both `SP | CP` and treats them equally. hence, this change is made here to ensure that the behavior is identical (but an issue has been raised so that the frontend will call when `authType === CP`)!

Reply:
as commented elsewhere, I don't think we should support corppass for this